### PR TITLE
Added views for Benchmarks, Provisioning and OpenHPC

### PR DESCRIPTION
**Pull request by BaptisteGerondeau**
Default view for users is still the all view though.
This needs to be changed on a user basis.

### CODE REVIEW

File: files/views/provisioning_view.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+- view:
+        name: Provisioning 
+        view-type: list
+        description: "This page allows you to select which of the HPC Lab's machine/cluster you want to provision  "
+        filter-executors: true
+        filter-queue: true
+        job-name:
+            - os_provision
+            - cluster_provision
```
[the remark this answers is not in the document]
Comment (BaptisteGerondeau) [reply files/views/provisioning_view.yml]: provisioning_job is the lower level of the jobs, it shouldn't be used by users (or only for debug, in which case you want to have the machine_provision as well).
I was thinking of the views as separating "generic use" from "debug", and hiding the machine and provisioning job is part of that.

File: files/views/ohpc_view.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+- view:
+        name: OpenHPC 
+        view-type: list
+        description: "This page allows you to select which OpenHPC-related job to run on the HPC Lab's clusters  "
+        filter-executors: true
+        filter-queue: true
+        job-name:
+            - cluster_provision
```
[the remark this answers is not in the document]
Comment (BaptisteGerondeau) [reply files/views/ohpc_view.yml]: You're right, both out !

File: files/views/provisioning_view.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (rengolin) [reply files/views/provisioning_view.yml]: Ok, just the two is fine, if neither are in the other two groups.

**Comment by rengolin** [comment]
Had to fix a few trailing white space on views names. Next time, please check them with "git diff" or "git show", they should appear in bright red.

**Pull request merged**